app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,58 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field my-5">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: 'Username', class: 'border my-3 py-1 px-3' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autocomplete: "email" %>
+  </div>
+
+  <div class="field my-5">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio, autocomplete: "bio", placeholder: 'Bio', class: 'border my-3 py-1 px-3' %>
+  </div>
+
+  <div class="field my-5">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo, autocomplete: "photo", placeholder: 'Photo URL', class: 'border my-3 py-1 px-3' %>
+  </div>
+
+  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+    <% if @minimum_password_length %>
+      <br />
+      <em><%= @minimum_password_length %> characters minimum</em>
+    <% end %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
+    <%= f.password_field :current_password, autocomplete: "current-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Update" %>
+  </div>
+<% end %>
+
+<h3>Cancel my account</h3>
+
+<div>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?", turbo_confirm: "Are you sure?" }, method: :delete %></div>
+
```

Comment:
- [ ] For some reasons I can't decipher, I am unable to log in after signing up successfully as shown below. Kindly check your `login`/`views` views to ensure that the `link` or `render` is implemented correctly.

https://watch.screencastify.com/v/P3M1LJF9OzEXFS7oaY9Y

  - Below screenshot affirms that users where signed in signed up successfully :point_down: :point_down: 
![Screenshot from 2023-10-13 15-40-46](https://github.com/ClarenceAtim/Blog-App-/assets/99715640/191540f1-ec6f-4089-88fb-ec61676e9b0d)

Reply:
Hi @AbayomiOlaoye ,

I am able to log in successfully after confirming the email.

![image](https://github.com/ClarenceAtim/Blog-App-/assets/125105700/4cefd3ea-335f-4deb-8de4-d0804d8c49ed)

Reply:
Hi @ClarenceAtim , now that's cool but there was no pop-up for confirmation nor any email was received to confirm the mail. I think it's pointing to a user experience issue there which you might want to consider too, by handling errors messages in a more clear and understandable way for users.

If everything checks out now, please submit again 👏  and congratulations 🎉